pkg/auth/round_trippers_test.go

```diff
@@ -76,12 +87,13 @@ func TestImpersonatingRoundTripper(t *testing.T) {
 			rt := &testRoundTripper{}
 			_, err := NewImpersonatingRoundTripper(rt).RoundTrip(req)
 			require.NoError(t, err)
-			if ts.expected == "" {
+			if ts.expectedUser == "" {
 				_, ok := rt.Request.Header[transport.ImpersonateUserHeader]
 				require.False(t, ok)
 				return
 			}
-			require.Equal(t, ts.expected, rt.Request.Header.Get(transport.ImpersonateUserHeader))
+			require.Equal(t, ts.expectedUser, rt.Request.Header.Get(transport.ImpersonateUserHeader))
+			require.Equal(t, ts.expectedGroup, rt.Request.Header.Values(transport.ImpersonateGroupHeader))
```

Comment:
It would be great if we check group values even if the `expectedUser` is blank for [this](https://github.com/oam-dev/kubevela/pull/3713/files#diff-d23fc29a3ee18f5d0c238a21f6cfe8769843e53a3b5108b1920b0249a59115bdR66-R67) test case.

Reply:
Good idea. Another test is added. The current test is to verify empty application in ContextWithInfo as well.